suite-native/module-home/src/screens/HomeScreen/useShowViewOnlyAlert.tsx

```diff
@@ -109,6 +110,7 @@ export const useShowViewOnlyAlert = () => {
             !isPortfolioTrackerDevice &&
             !hasDiscovery &&
             !viewOnlyCancelationTimestamp &&
+            !isCreatingNewPassphraseWallet &&
```

Comment:
I like how the canBeShowed condition is super short and simple 🤣

Reply:
I think we are getting closer to the moment we were talking about: having one decision tree to show modals.